symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -70,15 +70,15 @@ public ApiClientBuilderJersey2() {
   public ApiClient build() {
```

Comment:
I'm wondering if we are not opening too much the builder/client and if it would not be enough to just let the user override the build() method (vs opening createClientConfig, configureProxy, configureJackson,...)